docs/contacts/activity/logging-calls.md

```diff
@@ -0,0 +1,64 @@
+# Hubspot: How to log a call
+
+!!! Note
+   The purpose of the CRM is to have a centralized location for all communication with agencies we work with. Logging calls is an important way to track communication.
```

Comment:
The text here (The purpose of the CRM is to have a centralized location for all communication with agencies we work with. Logging calls is an important way to track communication.) is not inside the Note box. Not sure it needs to be but as it is now, there's an empty Note box